basic/src/test/java/org/jboss/hal/testsuite/test/configuration/undertow/HTTPSListenerTestCase.java

```diff
@@ -363,7 +365,17 @@ public void editSendBufferInvalid() throws Exception {
 
     @Test
     public void editSocketBinding() throws Exception {
-        editTextAndVerify(HTTPS_LISTENER_ADDRESS, SOCKET_BINDING, undertowOps.createSocketBindingWithoutReference());
+        String socketBinding = undertowOps.createSocketBinding();
+        ModelNodeResult result = operations.readAttribute(HTTPS_LISTENER_ADDRESS, SOCKET_BINDING);
```

Comment:
In that case the cleanup logic has to be more complicated and harder to read IMO since it should either write or undefine attribute depending on the result of the opening read operation. Is it worth it?